migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,426 @@
+
+[NOTE]
+====
+The former `settings.xml` are also called global settings, the latter `settings.xml` are referred to as user settings.
+If both files exists, their contents gets merged, with the user-specific `settings.xml` being dominant.
+====
+
+[TIP] 
+====
+If you need to create user-specific settings from scratch, it's easiest to copy the global settings from your Maven installation to your *_${user.home}/.m2_* directory.
+Maven's default *_settings.xml_* is a template with comments and examples so you can quickly tweak it to match your needs.
+_If you are looking to learn more about how to manage your various Maven settings, please visit the https://maven.apache.org/settings.html[Maven Settings Reference]._
+====
+
+{PRODUCT_LONG} v{VERSION_OLD} product dependencies are hosted by {RH_MAVEN_URL_BASE}[Red Hat's Maven repository], so you need to ensure that your `settings.xml` reflects that, as seen in the following example:
+
+include::../partials/settings-redhat-maven.adoc[]
+
+=== Update to use the Maven Bill of Materials (BOM)
+
+A bill of materials (BOM) is a list of the parts or components that are required to build a product.
+BOMs are ordinary `pom.xml` files — they contain no source code and their only purpose is to declare their bundled modules.
+
+.Always use the Maven Bill-of-Materials (BOM)
+[IMPORTANT]
+====
+It is highly recommended to use a bill of materials (BOM) to manage project depencencies, as it simplifies the dependency management by organizing a group of depdenencies under a single version. 
+====
+
+A bill of materails (BOM) can be added to an existing POM file by adding it to the **_dependencyManagement_** section as a dependency with a pom type:
+
+include::../partials/maven-bom-now.adoc[]
+
+When a bill of materials (BOM) is used it is no longer necessary to specify the depenency's version, as that comes from the BOM, so your depenencies end up looking like this:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+    </dependency>
+</dependencies>
+----
+
+[NOTE]
+====
+If, during the course of your Maven dependency update, you encounter any dependencies that are *not* included in the bill of materials (BOM), then use the Maven Library Version noted in the table below, such as:
+====
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+        <version>7.67.2.Final-redhat-00006</version>
+    </dependency>
+</dependencies>
+----
+
+== Product Version Mapping
+
+{PRODUCT_LONG} v{VERSION_OLD} is essentially a blue-washed version of the former Red Hat 7.13.x offering:
+
+The following table is useful in determining which community release is bound to is section details the mapping of https://www.kie.org/[KIE Community] releases to either the former Red Hat enterprise release or the current {PRODUCT_LONG} enterprise release.
```

Comment:
there's no direct relation between Red Hat downstream version and the community versions... it's somewhat misleading try to make this reference.